xarray/core/groupers.py

```diff
@@ -226,7 +286,51 @@ def factorize(self, group) -> EncodedGroups:
 
 @dataclass
 class TimeResampler(Resampler):
-    """Grouper object specialized to resampling the time coordinate."""
+    """
+    Grouper object specialized to resampling the time coordinate.
+
+    Attributes
+    ----------
+    freq : str
+        Frequency to resample to. See `Pandas frequency
+        aliases <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`_
+        for a list of possible values.
+    closed : {"left", "right"}, optional
+        Side of each interval to treat as closed.
+    label : {"left", "right"}, optional
+        Side of each interval to use for labeling.
+    base : int, optional
+        For frequencies that evenly subdivide 1 day, the "origin" of the
+        aggregated intervals. For example, for "24H" frequency, base could
+        range from 0 through 23.
+
+        .. deprecated:: 2023.03.0
+            Following pandas, the ``base`` parameter is deprecated in favor
+            of the ``origin`` and ``offset`` parameters, and will be removed
+            in a future version of xarray.
+
+    origin : {'epoch', 'start', 'start_day', 'end', 'end_day'}, pandas.Timestamp, datetime.datetime, numpy.datetime64, or cftime.datetime, default 'start_day'
```

Comment:
Here we mix `"` and `'` conventions.

Reply:
we're also missing a colon after `default` (should be `default: "start_day"`)